src/types/params/index.ts

```diff
@@ -2,3 +2,5 @@ export interface PaginationParams {
   page?: number
   size?: number
 }
+
+export type Language = "de" | "en" | "fr" | "it"
```

Comment:
I would not export this form the api client and use it as an internal interface as the package should not defined languages or name it specifically for the use case as we use Lanuges for the applications language support....

Reply:
It is exported internally, not externally. I've seen we typed the language in some calls and accepted strings in another, went with the concrete type.

Reply:
Ah nvm messed that up as the file is called index....